templates/user/settings/profile.tmpl

```diff
@@ -12,7 +12,7 @@
 						<span class="text red tw-hidden" id="name-change-prompt"> {{ctx.Locale.Tr "settings.change_username_prompt"}}</span>
 						<span class="text red tw-hidden" id="name-change-redirect-prompt"> {{ctx.Locale.Tr "settings.change_username_redirect_prompt"}}</span>
 					</label>
-					<input id="username" name="name" value="{{.SignedUser.Name}}" data-name="{{.SignedUser.Name}}" autofocus required {{if or (not .SignedUser.IsLocal) ($.UserDisabledFeatures.Contains "change_username") .IsReverseProxy}}disabled{{end}} maxlength="40">
+					<input id="username" name="name" value="{{.SignedUser.Name}}" data-name="{{.SignedUser.Name}}" data-global-init="initInputAutoFocusEnd" required {{if or (not .SignedUser.IsLocal) ($.UserDisabledFeatures.Contains "change_username") .IsReverseProxy}}disabled{{end}} maxlength="40">
```

Comment:
Why you assume that when a user goes to the "profile" page, they are highly likely to change their username?

If they don't want to change their username, why the input should be auto-focused?

Reply:
The starting point is as follows:
I was about to edit my username. When I came to this page, the input box for the username had already gained focus because the `autofocus` attribute was added to the `input` element.
Then:
① I had to click at the end of the `input` box, press the `Backspace` key to delete the original username, and then enter a new one.
② Or, I could press the `Delete` key to delete the original username and then enter a new one.
At least in my opinion, ordinary people use the `Backspace` key far more frequently than the `Delete` key. So I thought it was unreasonable, and that's why I naturally submitted this PR.

Reply:
I have no interest to do more arguments.

In short: GitHub doesn't do so, Discord doesn't do so, I guess GitLab doesn't do so either. 

I do not see any website do so.

Reply:
![after](https://github.com/user-attachments/assets/53991896-684b-473a-ae1e-e25c135a1fd9)

Reply:
1. It is NOT username
2. It is NOT autofocus or initInputAutoFocusEnd. They WILL NOT WORK because the element is hidden when the page loads.

Reply:
The screenshot of GitHub is RIGHT because when a user click "Edit Profile", they ARE HIGHLY LIKELY going to edit their name.

But as I said above: Gitea's most "autofocus" usages are abuses. 

> Why you assume that when a user goes to the "settings profile" page, they are highly likely going to change their username?